Deploy/stacks/dynamic/examples/datasets/inputs/config/building-cambridge.json

```diff
@@ -15,21 +15,21 @@
                     ]
                 }
             },
-            "columnMap": {
-                "IDval": "os_topo_toid",
-                "IDname": "os_topo_toid",
-                "footprint": "polygon",
-                "elevation": "abshmin",
-                "height": "relh2"
-            },
-            "importOptions": {
-                "sridIn": "27700"
-            },
-            "name": "cambridge_gsl",
-            "geoServerSettings": {
+			"columnMap": {
+				"IDval": "os_topo_toid",
+				"IDname": "os_topo_toid",
+				"footprint": "polygon",
+				"elevation": "abshmin",
+				"height": "relh2"
+			},
+			"importOptions": {
+				"sridIn": "27700"
+			},
+			"name": "building-cambridge_gsl",
+			"geoServerSettings": {
```

Comment:
Please configure whichever IDE you're using to replace tabs with spaces.

Reply:
I have now re-formatted changed files to have spaces instead of tabs.